pkg/rtnl/templates/list.html

```diff
@@ -0,0 +1,8 @@
+{{ template "base" . }}
+{{ define "content" }}
+
+<section class="text-center py-14" hx-get="/v1/links" hx-trigger="load">
```

Comment:
Ok, here is how to get around the authentication thing; just create an empty page that uses htmx and the `hx-trigger="load"` to fetch the content from the server.